compose/docker-compose.dev.yml

```diff
@@ -51,3 +53,19 @@ services:
   phpfpm:
     env_file: env/magento.env
     volumes: *appvolumes
+
+  ## Adding Elastic Search to run integration tests in local dev environment
+  elastic:
```

Comment:
Should we comment this out by default and describe in documentation that it should be uncommented is using integration tests? This would help save docker resource usage.

Reply:
Agree.

Reply:
done.